cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java

```diff
@@ -0,0 +1,48 @@
+package cwms.cda.api.enums;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jdk.nashorn.internal.runtime.Version;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.time.ZonedDateTime;
+
+@Schema(
+        name = "Version Type",
+        description = "Version type specifies the type of timeseries response to be received. Can be max aggregate "
+                + "or single version. Max aggregate cannot be ran if version date field is specified."
```

Comment:
"cannot be ran" doesn't sound right.

Reply:
Reddit, at least, agrees: https://www.reddit.com/r/grammar/comments/aefv2u/can_be_ran_vs_can_be_run/

Reply:
done